src/main/java/com/jcabi/dynamodb/maven/plugin/AbstractDynamoMojo.java

```diff
@@ -120,11 +122,20 @@ protected int tcpPort() {
     }
 
     /**
+     * Set the project environment.
+     * {@link com.jcabi.dynamodb.maven.plugin.AbstractEnviromentMojo}.
+     * @throws MojoFailureException If fails
+     */
+    protected void environment() throws MojoFailureException {
```

Comment:
If you make ```#environment``` to be an abstract method, then other sub classes ```com.jcabi.dynamodb.maven.plugin.StopMojo``` and ```com.jcabi.dynamodb.maven.plugin.CreateTablesMojo``` MUST implement it.
This is not what we want because these sub classes do not need to set up an environment for them. Only the sub classes that need environment implement it.